rental.rb

```diff
@@ -0,0 +1,11 @@
+class Rental
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
+    book.rentals.push(self)
+    person.rentals.push(self)
+  end
+  attr_accessor :date
+  attr_reader :book, :person
+end
```

Comment:
- [ ] Kindly make these attributes have setter and getter methods. `attr_accessor :book, :person`

Reply:
In that case I cannot use attr_accessor, I would have to write the setter so that it updates the book's and person's rentals. I would think that's a mess, because the previous rental would have to be removed, and a new rental be created. I will put the attr_accessor as you suggested but I definitely think it's not a good idea.

Reply:
Hi @acolombo1, I think you are mistaken. In Ruby, data is private in classes and that means you need to have methods for reading and updating/writing such attributes. The attr_accessor is the idiomatic way of reading and writing the data. Think of it this way, if you make a mistake when inputting the person who rented a book or the name of the book that was rented, how will you update it without a setter method? You can read this [article](https://www.educative.io/answers/what-is-attraccessor-in-ruby) to have a clearer understanding. 👍